application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -305,7 +308,7 @@ public function checkStageInForm(Validation $validation, $completed_stages, $ful
 	 */
 	public function checkRequiredStages(Validation $validation, $completed_stages, $fullData)
```

Comment:
I need to verify if this runs every time, or only when completed_stages changes. The latter would be broken.